File: core/data/tests/kebab_case_rename/output.swift
import Foundation

/// This is a comment.
public struct Things: Codable {
	public let bla: String
	public let label: String?
	public let label_left: String?

	enum CodingKeys: String, CodingKey, Codable {
		case bla,
			label,
			label_left = "label-left"
	}

	public init(bla: String, label: String?, label_left: String?) {
		self.bla = bla
		self.label = label
		self.label_left = label_left
	}
}

File: core/data/tests/kebab_case_rename/output.kt
@file:NoLiveLiterals

package com.agilebits.onepassword

import androidx.compose.runtime.NoLiveLiterals
import kotlinx.serialization.*

/// This is a comment.
@Serializable
data class Things (
	@SerialName("bla")
	val bla: String,
	@SerialName("label")
	val label: String? = null,
	@SerialName("label-left")
	val label_left: String? = null
)

